packages/SketchMorph2-Tests.package/M2DrawingComponentTests.class/instance/testSimpleTextAfterMovingByHandle.st

```diff
@@ -0,0 +1,10 @@
+testing
+testSimpleTextAfterMovingByHandle
+	self viewPort
+		addText;
+		clickAt: 75 asPoint.
+	self viewPort temporaryTextMorph contents: 'test'.
+	self viewPort temporaryTextMorph moveByHandleTo: 25 asPoint - ((0.5@1) * self viewPort temporaryTextMorph handle extent).
+	self viewPort temporaryTextMorph position: self viewPort temporaryTextMorph topCenter.
```

Comment:
You are setting the position by moving the handle in the first line and overwriting it in the second, if I understand correctly. Doesn't it render the first line useless?